pkg/manager/internal/controllers/organizationroletemplate_controller.go

```diff
@@ -66,23 +67,59 @@ func (r *OrganizationRoleTemplateReconciler) Reconcile(req ctrl.Request) (ctrl.R
 		return ctrl.Result{}, nil
 	}
 
+	var targets []corev1alpha1.OrganizationRoleTemplateTarget
 	organizations := &storagev1alpha1.OrganizationList{}
 	if err := r.Client.List(ctx, organizations); err != nil {
 		return ctrl.Result{}, fmt.Errorf("listing Organizations: %w", err)
 	}
 
-	var targets []corev1alpha1.OrganizationRoleTemplateTarget
-	for _, organization := range organizations.Items {
-		if organization.IsReady() {
+	// Reconcile Role/RoleBindings in Organization namespaces.
+	if organizationRoleTemplate.HasScope(corev1alpha1.RoleTemplateScopeOrganization) {
+		for _, organization := range organizations.Items {
+			if !organization.IsReady() {
+				// skip Unready Organizations.
+				continue
+			}
 			if err := r.reconcileRBACForOrganization(ctx, organizationRoleTemplate, &organization); err != nil {
-				return ctrl.Result{}, fmt.Errorf("reconcling Organization Role: %w", err)
+				return ctrl.Result{}, fmt.Errorf("reconcling Organization RBAC: %w", err)
 			}
 			targets = append(targets, corev1alpha1.OrganizationRoleTemplateTarget{
 				Kind:               organization.Kind,
-				APIGroup:           "storage.bulward.io",
+				APIGroup:           organization.GroupVersionKind().Group,
```

Comment:
Are you sure this works? Since you'd usually use `apiutil.GVKForObject` to get the GVK. Type metadata is usually empty in go types IIRC

Reply:
ok